AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.h

```diff
@@ -16,7 +16,6 @@
 #define SERVER_REGION "YOUR_SERVER_REGION"
 
 @property (weak, nonatomic) IBOutlet UIButton *joinButton;
-@property (weak, nonatomic) IBOutlet UIButton *leaveButton;
```

Comment:
Little confused, we have two views, but one view controller?

Reply:
Yeah, that's how the UI constructed currently, don't want to make dramatic changes since it's not related to this PR, it's also common to have more than 1 container views under one controller.